sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -240,7 +240,7 @@ int gnrc_ipv6_nib_get_next_hop_l2addr(const ipv6_addr_t *dst,
                 gnrc_netif_acquire(netif);
             }
             node = _nib_onl_get(&route.next_hop,
-                                (netif == NULL) ? netif->pid : 0);
+                                (netif != NULL) ? netif->pid : 0);
```

Comment:
This was an actual bug waiting to happen?

Reply:
I guess we were lucky (or rather unlucky) that this line never really was hit.